api/templates/teledeclaration_pdf.html

```diff
@@ -1,714 +1,975 @@
 {% load static %}
 <html>
-    <head>
-        <style>
-            @font-face {
-            font-family: 'Marianne';
-            src: url({% static '../../web/static/fonts/Marianne-Regular.ttf' %}) format('truetype');
-            }
-            @font-face {
-                font-family: 'Marianne-Bold';
-                src:  url({% static '../../web/static/fonts/Marianne-Bold.ttf' %}) format('truetype');
-            }
-            @font-face {
-                font-family: 'Marianne-ExtraBold';
-                src: url({% static '../../web/static/fonts/Marianne-ExtraBold.ttf' %}) format('truetype');
-            }
-            body {
-                font-family: 'Marianne', Arial, Helvetica, sans-serif;
-            }
-            h1 {
-                font-family: 'Marianne-ExtraBold';
-                font-size: 24px;
-                margin: 32px 0px 16px 0px;
-            }
-            h2 {
-                font-family: 'Marianne-Bold';
-                font-size: 16px;
-                margin: 16px 0px;
-            }
-            h3 {
-                font-family: 'Marianne-Bold';
-            }
-            li span {
-                font-family: 'Marianne-Bold';
-            }
-            p, li {
-                margin: 16px 0px;
-            }
-            .image-container {
-                background-color:#0c7f46;
-                padding: 16px;
-            }
-            .other-measures > div {
-                margin-bottom: 10px;
-            }
-            .other-measures > div > span {
-                margin-right: 4px;
-            }
-            hr {
-                margin-top: 24px;
-            }
-        </style>
-    </head>
-    <body>
-        <div class="image-container">
-            <img src="{% static '../../web/static/images/logo_transparent_white.png' %}" alt="ma cantine" height="30" width="142" />
-        </div>
-        <h1>Justificatif pour télédéclaration {{ year }}</h1>
-        <h2>Cantine {{ canteen_name }} (SIRET {{ siret }})</h2>
-        <h3>Télédéclaration faite le {{ date|date:"l j F Y" }} par {{ applicant }}</h3>
-        {% if teledeclaration_mode != "SATELLITE_WITHOUT_APPRO" %}
-            <h3>
-                Vous avez choisi la télédéclaration {{ diagnostic_type }}.
-            </h3>
-        {% endif %}
+  <head>
+    <style>
+      @font-face {
+        font-family: "Marianne";
+        src: url({%static"../../web/static/fonts/Marianne-Regular.ttf"%}) format("truetype");
```

Comment:
Je crois qu'il faut un espace entre `static` et le path, d'où la build qui échoue

Reply:
😬 j'ai clairement les mauvaises parametres niveau VSCode. I'll revert the changes to this file